plugins/hydra_submitit_launcher/README.md

```diff
@@ -0,0 +1 @@
+# Hydra Submitit plugin
```

Comment:
Make a pass comparing this plugin name, package name module names etc to the other plugins to be sure it's consistent.